docs/_docs/02_features/engine.md

```diff
@@ -67,7 +67,7 @@ Parameters:
 
 * `source`: (Required) The source of the plugin. Multiple engine approaches are supported, including GitHub repositories, HTTP(S) paths, and local absolute paths.
 * `version`: (Required for GitHub) The version of the plugin to download from GitHub releases.
-* `type`: (Required) Currently, only `rpc` type is supported.
+* `type`: Currently, the only supported type is `rpc`.
```

Comment:
I was saying that we should be able to make the `type` attribute of the `engine` configuration block optional, with a default value of `rpc`, right?

Reply:
Yes, type is optional